.jshintrc

```diff
@@ -10,8 +10,20 @@
   "strict": true,
   "undef": true,
   "unused": true,
+  "expr": true,
   "globals": {
     "angular": false,
-    "prettyPrintOne": false
+    "prettyPrintOne": false,
+    // MOCHA
```

Comment:
And from the official jshintrc repo also: https://github.com/jshint/jshint/blob/master/examples/.jshintrc